dowhy/causal_model.py

```diff
@@ -308,26 +308,17 @@ def estimate_effect(
                 causal_estimator = self._estimator_cache[method_name]
             else:
                 causal_estimator = causal_estimator_class(
-                    self._data,
                     identified_estimand,
-                    self._treatment,
-                    self._outcome,  # names of treatment and outcome
-                    control_value=control_value,
-                    treatment_value=treatment_value,
-                    test_significance=test_significance,
-                    evaluate_effect_strength=evaluate_effect_strength,
-                    confidence_intervals=confidence_intervals,
-                    target_units=target_units,
-                    effect_modifiers=effect_modifiers,
                     **method_params,
```

Comment:
If I understand correctly, we do not pass the actual params here because we expect the `estimate_effect` call to take care of it, right?

Reply:
Yep, here we are initializing the estimator, just creating it, then we need to call `fit()` and then `estimate_effect()`